dev_env_setup.sh

```diff
@@ -12,145 +12,75 @@ cd "$SCRIPT_DIR"
-    done
-
-    # Create new project using cubejs-cli
-    echo "Creating new project with name $APP_NAME and database type $DB_TYPE..."
-    npx cubejs-cli create "$APP_NAME" -d "$DB_TYPE"
-
-    # Step 11: Run yarn link @cubejs-backend/server-core in your project directory
-    echo "Linking @cubejs-backend/server-core in the project directory..."
-    cd "$APP_NAME"
-    yarn link @cubejs-backend/server-core
-    cd ../
-else
-    echo "Ok. No problem!"
-    echo "You need to run 'yarn link @cubejs-backend/server-core' in your project directory manually"
-fi
-
-# Step 11: Ask user if they plan to make changes to Rust code
-read -p "Do you plan to make changes to Rust code? (yes/no, default: no): " RUST_CHANGES
-RUST_CHANGES=${RUST_CHANGES:-no}
+# Create new project using cubejs-cli
+echo "Creating new project with name $APP_NAME and database type $DB_TYPE..."
+node "$SCRIPT_DIR/packages/cubejs-cli/dist/src/index.js" create "$APP_NAME" -d "$DB_TYPE"
 
-if [[ "$RUST_CHANGES" == "yes" || "$RUST_CHANGES" == "y" ]]; then
-    # Run yarn link:dev in the script directory
-    cd "$SCRIPT_DIR"
-    echo "Running 'yarn link:dev' in the root directory..."
-    yarn link:dev
+# Step 5: Link all packages in the new project
+echo "Linking packages in the new project..."
+cd "$APP_NAME"
 
-    if [[ "$CREATE_PROJECT" == "yes" || "$CREATE_PROJECT" == "y" ]]; then
-        dev_pkgs=("@cubejs-backend/shared"
-                  "@cubejs-backend/cloud"
-                  "@cubejs-backend/native"
-                  "@cubejs-backend/server"
-                  "@cubejs-backend/server-core"
-                  "@cubejs-backend/api-gateway"
-                  "@cubejs-backend/schema-compiler"
-                  "@cubejs-backend/query-orchestrator"
-                  "@cubejs-backend/athena-driver"
-                  "@cubejs-backend/duckdb-driver"
-                  "@cubejs-backend/bigquery-driver"
-                  "@cubejs-backend/postgres-driver"
-                  "@cubejs-backend/databricks-jdbc-driver"
-                  "@cubejs-backend/mssql-driver"
-                  "@cubejs-backend/clickhouse-driver"
-                  "@cubejs-backend/snowflake-driver"
-                  "@cubejs-backend/cubestore-driver"
-                  "@cubejs-backend/templates"
-                  "@cubejs-client/core"
-                  "@cubejs-client/ws-transport"
-                  "@cubejs-client/playground"
-                  )
-
-        cd "$CURRENT_DIR/$APP_NAME"
-        echo "Linking dev packages in $APP_NAME project..."
-
-        for pkg in "${dev_pkgs[@]}"
-        do
-            echo "Linking $pkg..."
-            yarn link "$pkg"
-        done
-    else
-        echo "Don't forget to link packages that you plan to modify inside your project!"
+for package in "$SCRIPT_DIR"/packages/*; do
+    if [ -d "$package" ]; then
+        package_name=$(node -p "require('$package/package.json').name")
+        echo "Linking $package_name..."
+        yarn link "$package_name"
```

Comment:
It's probably good to cleanup the scripts in root package.json to remove potential confusion for those who will find it and try to use but it is outdated and not actual anymore. Like `"link:dev"`